src/main/resources/assets/actuallyadditions/lang/en_US.lang

```diff
@@ -351,6 +351,7 @@ item.actuallyadditions.item_misc_knife_blade.name=Knife Blade
 item.actuallyadditions.item_misc_knife_handle.name=Knife Handle
 item.actuallyadditions.item_misc_black_quartz.name=Black Quartz
 item.actuallyadditions.item_misc_ring.name=Ring
+item.actuallyadditions.item_misc_ring.storage=%s/%s Blaze Stored
```

Comment:
I'd prefer just "Blaze Stored" being localized here, with the way it is formatted being hardcoded.